src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-size: 16px;
+  font-weight: 300;
+  padding: 0;
```

Comment:
let's reset all margin and padding like you did on the lesson

Reply:
fixed